.github/workflows/deploy.yml

```diff
@@ -3,9 +3,19 @@ name: Deploy to Gov PaaS
 on:
   workflow_dispatch:
     inputs:
-      deploy_to_dev:
+      environment:
+        description:  Which AWS Account to use
+        type: choice
+        required: true
+        options:
+        - test
+        - uat
+      copilot:
+        description: Whether to deploy
+        type: boolean
         required: false
         default: false
+      deploy_to_dev:
         type: boolean
         description: Deploy to dev?
```

Comment:
Can we specifically say "Deploy to cloudfoundry dev" here, just to differentiate it from the copilot section